### DISCUSSION TITLE
can someone provide a more elaborate explainer on how Again & skip really works?

### DISCUSSION THREAD

**Discussion by Daoet** [st3v3nmw/obsidian-spaced-repetition#864] [Q&A]
The manual stated again button sorta works like anki, but I don't think resetting the card comes close to again; I didn't see the card after my review session, however. And it raises some questions:

- How long does it take for me to see the affected card if I press again or skip?
- what is the difference between again and skip?
- do I need to hit again or hard when I fail to recall a card? How do things even work here?

**Comment by KyleKlus** [comment]
Hi, I think it is implemented like this:
- Reset: The card is move to the end of the queue and thus should be shown at the end of the session. This should reset the schedule, which this card originally had and thus the card is treated, as if it is a newly written, unviewed card.
- Skip: The card is removed from the review queue and thus won't be shown again in the current session 

That the card wasn't show could be a bug, or I didn't fully understand the review algorithm.
